homeassistant/components/sensor/rejseplanen.py

```diff
@@ -0,0 +1,226 @@
+"""
+Support for Rejseplanen information from rejseplanen.dk.
+
+For more info on the API see:
+https://help.rejseplanen.dk/hc/en-us/articles/214174465-Rejseplanen-s-API
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.rejseplanen/
+"""
+import logging
+from datetime import timedelta, datetime
+from operator import itemgetter
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+import homeassistant.util.dt as dt_util
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_NAME, ATTR_ATTRIBUTION
+from homeassistant.helpers.entity import Entity
+
+REQUIREMENTS = ['rjpl==0.3.2']
+_LOGGER = logging.getLogger(__name__)
+
+ATTR_STOP_ID = 'Stop ID'
+ATTR_STOP_NAME = 'Stop'
+ATTR_ROUTE = 'Route'
+ATTR_TYPE = 'Type'
+ATTR_DIRECTION = "Direction"
+ATTR_DUE_IN = 'Due in'
+ATTR_DUE_AT = 'Due at'
+ATTR_NEXT_UP = 'Later departure'
+
+CONF_ATTRIBUTION = "Data provided by rejseplanen.dk"
+CONF_STOP_ID = 'stop_id'
+CONF_ROUTE = 'route'
+CONF_DIRECTION = 'direction'
+CONF_DEPARTURE_TYPE = 'departure_type'
+
+DEFAULT_NAME = 'Next departure'
+ICON = 'mdi:bus'
+
+SCAN_INTERVAL = timedelta(minutes=1)
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_STOP_ID): cv.string,
+    vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+    vol.Optional(CONF_ROUTE, default=[]):
+        vol.All(cv.ensure_list, vol.Length(min=0)),
+    vol.Optional(CONF_DIRECTION, default=[]):
+        vol.All(cv.ensure_list, vol.Length(min=0)),
+    vol.Optional(CONF_DEPARTURE_TYPE, default=[]):
+        vol.All(cv.ensure_list, vol.Length(min=0))
+})
+
+
+def due_in_minutes(timestamp):
+    """Get the time in minutes from a timestamp.
+
+    The timestamp should be in the format day/month/year hour/minute/second
```

Comment:
The `strptime` call does not seem to match the documented format.

Reply:
Fixed.